atomic_reactor/util.py

```diff
@@ -670,12 +673,11 @@ def create_from_config(cls, workflow, registry=None, access=None):
                 raise RuntimeError(msg)
         else:
             # Try to match a registry in pull_registries, if no match found, use source_registry
```

Comment:
Comment will need to be updated (possibly docstring as well)

Reply:
updated